### PR TITLE
feat: Add max_quantity of ticket in order-statistics-event

### LINKED ISSUE #6710: Remove event-orga endpoint
It is not being used and redundant. Create proper sales stats API if needed

### PR DESCRIPTION
**Pull request by codedsun**
…ed order_statistics in event

Fixes #6710
feat: Add max_quantity of ticket in order-statistics-event

#### Changes proposed in this pull request:

-
-
-

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           

Complexity increasing per file
==============================
- app/api/order_statistics/events.py  1
- app/models/event.py  1
         

Clones added
============
- app/models/event.py  5
         

Clones removed
==============
+ app/api/order_statistics/events.py  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=4801455&bid=15979077)

File: app/api/order_statistics/events.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 from flask_rest_jsonapi import ResourceDetail
 from marshmallow_jsonapi import fields
-from marshmallow_jsonapi.flask import Schema
+from marshmallow_jsonapi.flask import Schema, Relationship
```
Comment (niranjan94) [inline app/api/order_statistics/events.py]: Codacy Issue found: ['marshmallow_jsonapi.flask.Relationship' imported but unused](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=4801455)

File: app/models/event.py [hunk 2]
```diff
@@ -24,6 +24,7 @@
 from app.models.ticket_fee import get_fee
 from app.models.ticket_fee import get_maximum_fee
 from app.models.ticket_holder import TicketHolder
+from app.models.order import Order, OrderTicket
```
Comment (niranjan94) [inline app/models/event.py]: Codacy Issue found: [redefinition of unused 'Order' from line 18](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=4801455)

Comment (niranjan94) [inline app/models/event.py]: Codacy Issue found: [Reimport 'Order' (imported line 18)](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=4801455)

File: app/api/order_statistics/events.py (hunk 1 above)
Comment (niranjan94) [inline app/api/order_statistics/events.py]: Codacy Issue found: [Unused Relationship imported from marshmallow_jsonapi.flask](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=4801455)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/api/order_statistics/events.py  1
- app/models/event.py  1
         

Clones added
============
- app/models/event.py  6
         

Clones removed
==============
+ app/api/order_statistics/events.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=4801455&bid=15979077)

File: app/api/order_statistics/events.py [hunk 3]
```diff
@@ -28,9 +29,13 @@ class Meta:
 
     id = fields.Str()
     identifier = fields.Str()
-    tickets = fields.Method("tickets_count")
-    orders = fields.Method("orders_count")
-    sales = fields.Method("sales_count")
+    tickets = fields.Str()
+    orders = fields.Str()
+    sales = fields.Str()
+    # tickets = fields.Method("tickets_count")
```
Comment (codedsun) [inline app/api/order_statistics/events.py]: @iamareebjamal - How do i make the `tickets` variable of dynamic type like 
for when I want to work with GET call of 
`http://127.0.0.1:5000/v1/users/8/events?page[size]=0&include=order_statistics`
the tickets should be of  type `fields.Dict()`

and when I call GET `http://127.0.0.1:5000/v1/events/38/order-statistics` it should be of 
type `fields.Method("tickets_count")`

Stucked here!

**Comment by iamareebjamal** [comment]
@codedsun Please complete this now

**Comment by codedsun** [comment]
@iamareebjamal - Done check

**Comment by iamareebjamal** [comment]
Does not filter deleted tickets out

**Comment by codedsun** [comment]
@iamareebjamal - Chck this

**Pull request merged**